huaweicloud/resource_huaweicloud_sfs_turbo.go

```diff
@@ -195,13 +239,41 @@ func buildTurboCreateOpts(cfg *config.Config, d *schema.ResourceData) shares.Cre
 	return result
 }
 
+func validateParameter(d *schema.ResourceData) error {
+	_, okHpcBandwidth := d.GetOk("hpc_bandwidth")
+	_, okHpcCacheBandwidth := d.GetOk("hpc_cache_bandwidth")
+	switch d.Get("share_type").(string) {
+	case shareTypeHpc:
+		if !okHpcBandwidth {
+			return fmt.Errorf("`hpc_bandwidth` is required when share type is HPC")
+		}
```

Comment:
shall we validate the case of HPC shared type and the HpcCacheBandwith is set, if so, pls check the follow case

Reply:
There has ConflictsWith for field `hpc_bandwidth`, `enhanced` and `hpc_cache_bandwidth`.

Reply:
okey